ntbs-service/Pages/SearchOld/Index.cshtml.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;

```

Comment:
Moved this file (and associated .cshtml file) to here so that they aren't deleted